ledger/common/hash/hash.go

```diff
@@ -13,17 +13,13 @@ type Hash [HashLen]byte
 var DummyHash Hash
 
 // HashLeaf returns the hash value for leaf nodes.
-//
-// path must be a 32 byte slice.
-// note that we don't include the keys here as they are already included in the path.
```

Comment:
The detail which I wanted to update is that `path` is not a `slice` anymore. Since `Hash` is directly defined above: https://github.com/onflow/flow-go/blob/5bc9d7bb00f9bb1808709d1a24ecc2fd61434bc7/ledger/common/hash/hash.go#L5-L9
there is literally no chance you can pass in an input of the wrong size ... so I was lazy and just removed the comment.  But you are right: better to mention it in the doGoc. 

👍 Updated the comment.